lib/export_tables_to_big_query.rb

```diff
@@ -0,0 +1,254 @@
+require 'fileutils'
```

Comment:
I preferred to have 2 services instead of just 1. One that exports to CloudStorage  and one that imports on BigQuery. 250 lines of code in one file is a lot, plus I do not see the `spec/lib.export_tables_to_big_query_spec` file.

Reply:
It performs the single task of loading files from our system to BigQuery.  The two phases required to do that are an implementation detail, subject to change, and one is entirely dependent on the other. It does not make sense to separate them into different files for the sole purpose of reducing class size.  It may be useful from a testing standpoint.  

Given that this is almost three sprints behind schedule, and the now-missing tests had very light and specific (read 'fragile') coverage, it did not make sense to try and adapt them; rewriting them to be more robust and provide higher coverage would have taken at least another week. 

I've put a ticket in JIRA to review the overall approach and tech debt and decide what our next steps need to be: https://dfedigital.atlassian.net/browse/TEVA-527

Are you willing to accept this as a compromise?

Reply:
Yes willing to accept the compromise, thanks